lib/Declarations.cpp

```diff
@@ -239,6 +224,50 @@ FunctionDecl::DeclareInModule(std::string_view name,
   return func;
 }
 
+size_t BasicBlockContext::GetParamIndex(const ParameterDecl &decl) const {
```

Comment:
Can this be an equality predicate on ParameterDecl? Ie. we create an operator== on ParameterDecl, then it's just std::find. Didnt we bump anvill to c++ 20, i think default equality operators on ParameterDecl should work right?